pkg/plugin/extension_builder_test.go

```diff
@@ -51,6 +52,13 @@ func TestExtensionBuilder_Visualizer(t *testing.T) {
 	res := b.ID("xxx").Visualizer("ccc").MustBuild()
 	assert.Equal(t, visualizer.Visualizer("ccc"), res.Visualizer())
 }
+func TestExtensionBuilder_WidgetLayout(t *testing.T) {
+	var b = NewExtension()
+	wl := &scene.WidgetLayout{Extendable: true}
+	wl2 := &scene.WidgetLayout{Extendable: true}
+	res := b.ID("xxx").WidgetLayout(wl).MustBuild()
+	assert.Equal(t, wl2, res.WidgetLayout())
```

Comment:
```suggestion
	assert.Same(t, wl2, res.WidgetLayout())
```

Use Equal and Same as needed; Same is needed for more stringent testing of identities; Equal will not detect them.